### ISSUE TITLE
Cycle barrier appearing giving selection for cars only

### ISSUE THREAD

**Issue by Bzeager** [streetcomplete/StreetComplete#4192]
![IMG_20220709_133319](https://user-images.githubusercontent.com/53423215/178090049-89f30b71-2d38-4896-a187-94ebff2ea7b6.jpg)
![IMG_20220709_133325](https://user-images.githubusercontent.com/53423215/178090054-65b82128-a557-4c36-813e-b71f660f7541.jpg)
![IMG_20220709_133730](https://user-images.githubusercontent.com/53423215/178090077-64628e2a-9761-4ccf-960f-b132cb446ff8.jpg)

**Comment by matkoniecz** [comment]
OK, that is quite weird.

I suspect that it has both `barrier=` and `traffic_calming=yes` tag.

Can you link area where this is happening?

**Comment by Bzeager** [comment]
@matkoniecz 
https://www.openstreetmap.org/node/1969381328/history

**Comment by FloEdelmann** [comment]
I guess it'd make sense to exclude nodes tagged with `cycle_barrier` in the traffic calming type quest, and remove any `traffic_calming` tags when answeting the cycle barrier type quest.

**Comment by matkoniecz** [comment]
I would be tempted to remove that `traffic_calming=yes` as invalid - it is blocking traffic (marked with barrier tag) rather than merely calming it...

Is such combination present anywhere else?

**Comment by Bzeager** [comment]
Sure, if traffic_calming is exclusively for highways as it looks on the Wiki page then yeah would make sense to remove and wouldn't be a bug. I try not get in the nomenclature debates, so outcome of this thread I'm not concerned with :) more just an FYI as something funny, as a potential bug, that got my attention.

**Comment by matkoniecz** [comment]
Definitely thanks for reporting it!

Traffic calming like bumps can exist on cycleways, but this specific combination looks wrong to me.

**Comment by FloEdelmann** [comment]
`traffic_calming` and `cycle_barrier` are used 85 times worldwide on the same node: https://overpass-turbo.eu/s/1k0h
`traffic_calming` and `barrier=cycle_barrier` are used 91 times worldwide on the same node: https://overpass-turbo.eu/s/1k0i

**Comment by westnordost** [comment]
I do not think it is StreetComplete's duty to "repair" such mistagging when it touches the element. I suppose there are many weird tagging combinations that don't quite fit but trying to clean up (automatically) such situations only because the element is touched by StreetComplete doesn't sound ok to me. We would put quite a lot of work onto ourselves plus of course there is always the danger that such automatisms (removing allegedly wrong tagging) is done in error or will lead to controversies.

I can at least think of theoretical situations where `traffic_calming=yes` + `barrier=cycle_barrier` could be tagged describing a real-world situation (e.g. when cycle track is not mapped separately and a traffic calming happens to be at the same place as a cycle barrier).

Anyway, what could be done is to not show the "traffic calming" quest when there is a `barrier` tagged on the same element, but on the other hand, this does not really solve the issue because after all, `traffic_calming=yes` could be also be set on a node of a cycleway without `barrier=cycle_barrier` being tagged on it also.

My opinion: Do nothing and let the (confused) user create a note in this case because there is a very high chance that (in that case) `traffic_calming=yes` was tagged in error.

**Comment by matkoniecz** [comment]
> My opinion: Do nothing and let the (confused) user create a note in this case because there is a very high chance that (in that case) traffic_calming=yes was tagged in error.

I also created notes at all `traffic_calming=yes` + `barrier=*` (and fixed some very obvious mistakes) asking people to verify tagging. See for example https://www.openstreetmap.org/note/3264955 https://www.openstreetmap.org/note/3264954

Thanks for reporting this, but it appears to be a tagging mistake in OSM.

It is also rare enough (40 cases worldwide) that it does not require special code to support it.

**Comment by FloEdelmann** [comment]
> 40 cases worldwide

I found more, see https://github.com/streetcomplete/StreetComplete/issues/4192#issuecomment-1179545386, but not that many (around 90 cases worldwide).

**Comment by matkoniecz** [comment]
Note that I checked solely `traffic_calming=yes` what excludes say `barrier=cycle_barrier` `traffic_calming=chicane` which seems mostly pointless and worth removing but not causing problems reported here.
